resources/js/components/slugs/Slug.js

```diff
@@ -56,16 +63,20 @@ export default class Slug {
     #createSynchronously() {
         const symbols = Statamic.$config.get('asciiReplaceExtraSymbols');
         const charmap = Statamic.$config.get('charmap');
+
         let custom = charmap[this.#language] ?? {};
         custom["'"] = ""; // Remove apostrophes in all languages
         custom["’"] = ""; // Remove smart single quotes
-        custom[" - "] = " "; // Prevent `Block - Hero` turning into `block_-_hero`
```

Comment:
Why'd this need to be removed?

Reply:
I think I was thinking it might not be needed after these changes...

However, on second thought, I'll add it back since not all slug/handle fields specify `replacements`. 👍